_data/navigation.yml

```diff
@@ -26,6 +26,8 @@ wiki:
   - title: Project Management
     url: /wiki/project-management/
     children:
+      - title: Drone Flight Permissions
+        url: /wiki/system-design-development/drone-flight-permissions/
```

Comment:
This is a good entry for Project Management, but doesn't match the file location. It should be `wiki/project-management/drone-flight-permissions/`

Reply:
Thank you, good catch. I made the fix as requested.